mcp/pacemaker.in

```diff
@@ -90,6 +91,13 @@ if [ x = "x$PCMK_STACK" -a -f "@sysconfdir@/cluster/cluster.conf" ]; then
     PCMK_STACK=cman
 fi
 
+# Check if there is a valid watchdog-device configured in sbd config
+if [ x != "x$SBD_WATCHDOG_DEV" -a "/dev/null" != "$SBD_WATCHDOG_DEV" -a -c "$SBD_WATCHDOG_DEV" ]; then
+	if chkconfig --list sbd_helper 2>/dev/null | grep -q ":on"; then
+		SBD_SERVICE=sbd_helper
+	fi
+fi
+
```

Comment:
I think we should guard this block with PCMK_STACK = "cman" since chkconfig is not universally available. There are ways to get cman without chkconfig, but that should be rare, and they're probably not using init. Alternatively, we can check whether chkconfig exists (which pacemaker_remote could use as well).